scenes/performance.gd

```diff
@@ -42,8 +43,11 @@ func print_song_loading_debug(to_print):
 		print(to_print)
 
 func _on_song_changed():
-	print("performance song changed")
 	current_song = SessionVariables.current_song
+	if(current_song is Song):
```

Comment:
No parentheses in if-statements unless if it is required in a boolean statement or used to clarify complex if-statements (check the style guide for examples).